src/query/storage/block.go

```diff
@@ -94,14 +138,36 @@ func (m multiSeriesBlock) SeriesMeta() []block.SeriesMeta {
 	return metas
 }
 
+func (m multiSeriesBlock) Consolidate() (block.Block, error) {
+	return &consolidatedBlock{
+		unconsolidated:    m,
+		consolidationFunc: block.TakeLast,
```

Comment:
We'll use a different consolidation func in the future, right?

Reply:
prometheus only has gauges so TakeLast is only reasonable option. Yeah, we would probably have different once we have different types.